code/datums/actions/xeno_action.dm

```diff
@@ -211,6 +217,9 @@
 
 //override this
 /datum/action/xeno_action/activable/proc/can_use_ability(atom/A, silent = FALSE, override_flags)
+	if(QDELETED(owner))
```

Comment:
if the owner has been qdeleted then you've got bigger issues, this is problem hiding.